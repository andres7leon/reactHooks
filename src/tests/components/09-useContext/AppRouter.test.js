import React from 'react';
import {mount} from 'enzyme'
import { AppRouter } from '../../../components/09-useContext/AppRouter';
import { UserContext } from '../../../components/09-useContext/UserContext';


describe('Pruebas componente AppRouter', () => {

    const user = {
        id: 1,
        name: 'andres'
    }

    const wrapper = mount( 
        <UserContext.Provider value={{user}}>
            <AppRouter />
        </UserContext.Provider>
    )

    test('debe de mostrarse bien ', () => {
        expect( wrapper ).toMatchSnapshot();
    })
    
    

});

