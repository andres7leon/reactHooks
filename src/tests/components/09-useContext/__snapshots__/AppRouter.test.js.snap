// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas componente AppRouter debe de mostrarse bien  1`] = `
<div>
  <nav>
    <ul>
      <li>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          className=""
          href="/About"
          onClick={[Function]}
        >
          About
        </a>
      </li>
      <li>
        <a
          className=""
          href="/Login"
          onClick={[Function]}
        >
          Login
        </a>
      </li>
    </ul>
  </nav>
  <div>
    <h1>
      HomeScreen
    </h1>
    <hr />
    <pre>
      {"id":1,"name":"andres"}
    </pre>
  </div>
</div>
`;
